pom.xml

```diff
@@ -160,7 +160,7 @@
     <reflections.version>0.9.11</reflections.version>
     <rocksdb.version>6.29.4.1</rocksdb.version>
     <shrinkwrap.version>3.0.1</shrinkwrap.version>
-    <slf4j.version>1.7.32</slf4j.version>
+    <slf4j.version>1.7.36</slf4j.version>
```

Comment:
that got me confused for a sec, why there is no new dependency added.
In case someone has the same question, this is because of 
> The SLF4J project has released version 1.7.35 containing the slf4j-reload4j module which supports reload4j. Moreover, the "slf4j-log4j12" artifact contains a Maven relocation directive for the "slf4j-reload4j" artifact.

https://reload4j.qos.ch/

Reply:
it was in the pull description 😢